src/modules/display.js

```diff
@@ -0,0 +1,29 @@
+import likeShow from './addLikes.js';
+
+export default function createShowCard(episode) {
+  const div = document.createElement('div');
+  div.classList.add('card');
+  div.setAttribute('id', episode.id);
+  div.innerHTML = `
+    <h2 class="card-title">${episode.name}</h2>
```

Comment:
- [ ] Same here,  Please, It's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions because you learned how to use ES6 new features. Kindly make this :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)

Reply:
Done.